lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -33,6 +33,7 @@
 import re
 import sys
 import yaml
+from collections import OrderedDict
```

Comment:
This *might* be a problem as `OrderedDict` originated in 2.7, but one of the shippable test versions is 2.6.

Reply:
It won't be.. I already used this to fix a bug in this file several months back when building nxapi requests.

Reference: https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/network/nxos/nxos.py#L304

Reply:
cool